Products/Poi/content/tracker.py

```diff
@@ -389,7 +389,7 @@ def getTagsInUse(self):
         issues = self.getIssues()
         tags = {}
         for i in issues:
-            for s in i.subject:
+            for s in i.Subject:
```

Comment:
If I recall, it was not pulling anything with lowercase. I think it is because it is using what is in the catalog index, which uses the capital S.